documentation/book/con-kafka-bridge-authentication.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-tls.adoc
+
+[id='con-kafka-bridge-authentication{context}']
+= Authentication support in Kafka Bridge
+
+Authentication is configured through the `authentication` property in `KafkaBridge.spec.kafka`.
+The `authentication` property specifies the type of the authentication mechanisms which should be used and additional configuration details depending on the mechanism.
+The currently supported authentication types are:
+
+* TLS client authentication
+* SASL-based authentication using the SCRAM-SHA-512 mechanism
+* SASL-based authentication using the PLAIN mechanism
+
+== TLS Client Authentication
+
+To use TLS client authentication, set the `type` property to the value `tls`.
+TLS client authentication uses a TLS certificate to authenticate.
+The certificate is specified in the `certificateAndKey` property and is always loaded from an {ProductPlatformName} secret.
+In the secret, the certificate must be stored in X509 format under two different keys: public and private.
+
+NOTE: TLS client authentication can be used only with TLS connections.
+For more details about TLS configuration in Kafka Bridge see xref:assembly-kafka-bridge-tls-{context}[].
+
+.An example TLS client authentication configuration
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersionPrev}
+kind: KafkaBridge
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  authentication:
+    type: tls
+    certificateAndKey:
+      secretName: my-secret
+      certificate: public.crt
+      key: private.key
+  # ...
+----
+
+== SCRAM-SHA-512 authentication
+
+To configure Kafka Bridge to use SASL-based SCRAM-SHA-512 authentication, set the `type` property to `scram-sha-512`.
+This authentication mechanism requires a username and password.
+
+* Specify the username in the `username` property.
+* In the `passwordSecret` property, specify a link to a `Secret` containing the password. The `secretName` property contains the name of the `Secret` and the `password` property contains the name of the key under which the password is stored inside the `Secret`.
+
+IMPORTANT: Do not specify the actual password in the `password` field.
+
+.An example SASL based SCRAM-SHA-512 client authentication configuration
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersionPrev}
+kind: KafkaBridge
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  authentication:
+  type: scram-sha-512
```

Comment:
I think the example is badly aligned. Everything bellow authentication should be two more spaces to the right?